lib/metasploit/framework/compiler/mingw.rb

```diff
@@ -34,6 +34,9 @@ def build_cmd(src)
 
           cmd << "#{self.mingw_bin} "
           cmd << "#{src_file} -I #{INCLUDE_DIR} "
+          if self.include_dirs
+            cmd << " #{self.include_dirs} "
+         cmd << "#{self.include_dirs.map { |include_dir| "-iquote #{include_dir}" }.join(' ')} " if self.include_dirs.any?
```

Comment:
Looks like Github has made a small mistake here; I think this should be:

```suggestion
          cmd << "#{self.include_dirs.map { |include_dir| "-iquote #{include_dir}" }.join(' ')} " if self.include_dirs.any?
```

Reply:
Yes ! thanks